cmake/modules/BuildDPDK.cmake

```diff
@@ -118,9 +118,8 @@ function(do_export_dpdk dpdk_dir)
 
   list(APPEND dpdk_components
     bus_pci
-    cmdline
+    telemetry
     eal
-    ethdev
```

Comment:
On newest dpdk there is no such libraries anymore. So this is the attempt to fix a linking error.

Reply:
In that case it probably makes sense to combine it with the change that bumps the spdk/dpdk submodule.

Reply:
since src/spdk submodule points to ceph/spdk.git repo (not original spdk) I want to get some help/feedback from maintainers how to bump the version correctly. Do I need to create another PR, etc.

Reply:
@rouming i will cherry-pick your change this change into #36775 as separated commit along with your change of src/blk/spdk/NVMEDevice.cc. i can take care of the cleanup removing the options from `src/common/legacy_config_opts.h` and `src/common/options.cc`,  and include it in your commit, if you are okay with it.

Reply:
Sure, will cook something.

Reply:
@rouming could you help review #36775 . i included/adapted your commits in it to ensure the change is atomic and compilable.